great_expectations/checkpoint/actions.py

```diff
@@ -387,8 +395,8 @@ def __init__(
 
     def _run(
         self,
-        validation_result_suite,
-        validation_result_suite_identifier,
+        validation_result_suite: ExpectationSuiteValidationResult,
+        validation_result_suite_identifier: ValidationResultIdentifier,
```

Comment:
@roblim -- Same as above: Do we need a `Union` with `GeCloudIdentifier` here?  Thanks.